evaluator/src/test/kotlin/ProjectSourceRuleTest.kt

```diff
@@ -142,3 +177,53 @@ private fun File.addDirs(vararg paths: String) {
         resolve(path).mkdirs()
     }
 }
+
+private fun ortResultWithDetectedLicenses(vararg detectedLicensesForFilePath: Pair<String, Set<String>>): OrtResult {
+    val id = Identifier("Maven:org.oss-review-toolkit:example:1.0")
+    val vcsInfo = VcsInfo(
+        type = VcsType.GIT,
+        url = "https://github.com/oss-review-toolkit/example.git",
+        revision = "0000000000000000000000000000000000000000"
+    )
+    val licenseFindings = detectedLicensesForFilePath.flatMapTo(sortedSetOf()) { (filepath, licenses) ->
+        licenses.map { license ->
+            LicenseFinding(license, TextLocation(filepath, startLine = 1, endLine = 2),)
```

Comment:
Please remove the trailing comma (also see the hint from the Qodana scan, which for some reason is associated to the last commit).